src/main/resources/db/changelog/v1.0/V1_001_create-student-table.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <include file="/db/changelog/v1.0/shared-properties.xml"/>
+
+    <changeSet id="create-student-table" author="polinatonko">
```

Comment:
I recommend to write id of changeset in the following way: `id="changeset-1.0.0"`. If it's required to define multiple changesets in one changelog, the you can follow this pattern:
```
id="changeset-1.0.0_001" - first changeset
id="changeset-1.0.0_002" - second changeset
...
```